.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: cpp
+git:
+  submodules: false
+addons:
+  ssh_known_hosts:
+  - gitlab.mpcdf.mpg.de
+  - simsgs.informatik.uni-stuttgart.de
+  apt:
+    packages:
+    - python2.7
+    - scons
+    - mpich 
+    - libmpich-dev
+before_install:
+  - "export PYTHONPATH=$PYTHONPATH:$(pwd)"
+  - travis_retry wget --quiet -O boost_1_65_1.tar.gz https://sourceforge.net/projects/boost/files/boost/1.65.1/boost_1_65_1.tar.gz/download
+  - tar xzvf boost_1_65_1.tar.gz >> log.txt
+  - cd boost_1_65_1
+  - ./bootstrap.sh 
+  - ./b2
+  - sudo ./b2 install
+  - ls
+  - cd ../../../../../..
+  - ls
+  - cd usr/local/lib
+  - ls
+  - cd ../../..
+  - cd /home/travis/build/SGpp/DisCoTec
+  - "export BOOST_LIBRARY_PATH=$BOOST_LIBRARY_PATH:/usr/local/lib"
+  - "export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib"
+  - "export BOOST_INCLUDE_PATH=$BOOST_INCLUDE_PATH:/usr/local/include/boost"
+script:
+  - sh ./compile.sh
+deploy:
+  github-token: $GITHUB_TOKEN
```

Comment:
One should also not just compile with the current configuration but also with other settings, such as ENABLE_FT=0/1 ISGENE=0/1 ... This does not need to be changed in this merge, but should be considered for the future so that we get a better test coverage of the possible compiler flag settings.